python/pyspark/sql/functions.py

```diff
@@ -101,19 +101,19 @@ def lit(col):
 
 
 @since(1.3)
-def col(col):
+def col(col_like):
     """
     Returns a :class:`~pyspark.sql.Column` based on the given column name.'
     """
-    return _invoke_function("col", col)
+    return _invoke_function("col", col_like)
 
 
 @since(1.3)
-def column(col):
```

Comment:
can we just change to direct assignment?

```
column = col
```

Also, it would be worth adding a small test in docstring.